src/modules/actions/components/moveTo.jsx

```
@@ -21,10 +21,13 @@ const moveTo = ({ t, canMove, pathname, navigate, isMobile, search }) => {
     icon,
     allowInfectedFiles: false,
     displayCondition: docs => {
+      // special case for rename in sharings tab
+      const isAllowedForSharedDrive = shouldHideIfSharedDriveRecipient
```

ESLint

[error] Replace `·shouldHideIfSharedDriveRecipient` with `⏎········shouldHideIfSharedDriveRecipient·&&`

(prettier/prettier)